Dev vulcan (#562)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/authentication/faq.mdx

---

> title:  Asked Questions
> sidebar_label:  Asked Questions

Reviewer: ```suggestion
title:  FAQ
sidebar_label:  FAQ
```

sidebar_position: 5
---

> ### Built-in account username is inconsistent with TAPTAP account username

Reviewer: ```suggestion
### The built-in account username is inconsistent with the TapTap account username
```

> Currently, TapTap nicknames and game nicknames are separate. Our design logic is that after using TapTap login for the first time, the login information will be stored in the built-in account. However, if you modify your TapTap nickname later, the original nickname stored in the built-in account will not be updated. If you want to use the updated nickname, you can try adding a [change nickname](/sdk/authentication/guide/#setting-other-user-properties) feature in the game. Users input their new nickname, and then the client calls the built-in account's method for setting other user attributes to update the nickname.

Reviewer: ```suggestion
Currently, TapTap nicknames and game nicknames are separate. Our design logic is that after using TapTap login for the first time, the login information will be stored in the built-in account. However, if you modify your TapTap nickname later, the original nickname stored in the built-in account will not be updated. If you want to use the updated nickname, you can try adding a [change nickname](/sdk/authentication/guide/#setting-other-user-properties) feature in the game. After a user provides their new nickname, the client calls the SDK method for setting other user attributes to update the nickname.
```

### 403 (Forbidden) error after changing password for built-in accounts

> If you have checked "Force client to re-login after password change" in Developer Center > Your Game > Game Services > Cloud Services > Built-in Accounts > Settings, then when a user changes their password, their session token will be reset. At this point, you need to prompt users to log in again; otherwise, they will encounter a 403 (Forbidden) error.

Reviewer: ```suggestion
If you have checked "Force client to re-login after password change" in Developer Center > Your Game > Game Services > Cloud Services > Built-in Accounts > Settings, then when a user changes their password, their session token will be reset. At this point, you need to prompt the user to log in again; otherwise, they will encounter a 403 (Forbidden) error.
```
